src/codec_svt.c

```diff
@@ -73,7 +73,9 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         }
         svt_config->encoder_color_format = color_format;
         svt_config->encoder_bit_depth = (uint8_t)image->depth;
+#if !SVT_CHECK_VERSION(0, 8, 8)
```

Comment:
Given the new SVT_AV1_CHECK_VERSION macro that @1480c1 added, I suggest the following.

1. After line 14 in this file, add our backup definition of that macro:

```
#ifndef SVT_AV1_CHECK_VERSION
#define SVT_AV1_CHECK_VERSION(major, minor, patch)                            \
    (SVT_AV1_VERSION_MAJOR > (major) ||                                       \
     (SVT_AV1_VERSION_MAJOR == (major) && SVT_AV1_VERSION_MINOR > (minor)) || \
     (SVT_AV1_VERSION_MAJOR == (major) && SVT_AV1_VERSION_MINOR == (minor) && \
      SVT_AV1_VERSION_PATCHLEVEL >= (patch)))
#endif
```

2. Change this line to
```
#if !SVT_AV1_CHECK_VERSION(0, 8, 8)
```

Reply:
Jamaika1: We need to add our backup definition of SVT_AV1_CHECK_VERSION as I suggested in my previous comment.

Please test this pull request with SVT-AV1 0.8.7.